### PR TITLE
[locale] vi: Improve short months

### PR DESCRIPTION
**Pull request by Serabe**
Short version for month is THG

@bangnk

### CODE REVIEW

**Comment by Serabe** [comment]
I don't know why changing `src/locale/vi.js` and `src/test/locale/vi.js` affects locale/ka tests. Does anybody can point me to the problem, please?

Thank you!

**Comment by marwahaha** [comment]
@Serabe - please pull from the latest code on `develop`. The tests should pass now.

**Comment by marwahaha** [comment]
@bangnk thoughts?

**Comment by Serabe** [comment]
Is there any way to look for a different person to check this?

**Comment by Serabe** [comment]
@marwahaha is there anyway to get a different person to check this.

**Comment by ichernev** [comment]
@Serabe yes, you can provide some official source of your claims (we love CLDR), it can also do the trick. The more official the better :)

**Comment by marwahaha** [comment]
this change makes sense. 
I found more agreement at CLDR at https://github.com/unicode-cldr/cldr-dates-modern/blob/master/main/vi/ca-gregorian.json#L16

@Serabe Thank you for adding this. Please add yourself as a locale author and resolve conflicts.

**Comment by marwahaha** [comment]
Any thoughts @bangnk ?

**Comment by Serabe** [comment]
Fixed conflicts, not adding myself as an author as I don't know Vietnamese to approve or disprove changes.

Thank you!

File: src/test/locale/vi.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
+import moment from '../../moment';
```
Comment (marwahaha) [inline src/test/locale/vi.js:1]: Do you need this line?

File: src/test/locale/vi.js [hunk 2]
```diff
@@ -27,14 +27,14 @@ test('parse', function (assert) {
...
         tests[i] = tests[i].split(',');
-        equalTest(tests[i][0], 'MMM', i);
-        equalTest(tests[i][1], 'MMM', i);
-        equalTest(tests[i][0], 'MMMM', i);
-        equalTest(tests[i][1], 'MMMM', i);
-        equalTest(tests[i][0].toLocaleLowerCase(), 'MMMM', i);
-        equalTest(tests[i][1].toLocaleLowerCase(), 'MMMM', i);
-        equalTest(tests[i][0].toLocaleUpperCase(), 'MMMM', i);
-        equalTest(tests[i][1].toLocaleUpperCase(), 'MMMM', i);
+        equalTest(tests[i][0], '[tháng] M', i);
```
Comment (marwahaha) [inline src/test/locale/vi.js:30]: Why did these change? The tests should pass without the test changes in `equalTest()`.

**Pull request merged**